build.gradle

```diff
@@ -174,20 +174,28 @@ test {
         exceptionFormat = 'full'
     }
 }
-downloadRobotServerPlugin {
-    version.set(remoteRobotVersion)
-}
 
-runIdeForUiTests {
-    systemProperty "robot-server.port", "8082"
-    systemProperty "ide.mac.message.dialogs.as.sheets", "false"
-    systemProperty "jb.privacy.policy.text", "<!--999.999-->"
-    systemProperty "jb.consents.confirmation.enabled", "false"
-    systemProperty "idea.trust.all.projects", "true"
-    systemProperty "ide.show.tips.on.startup.default.value", "false"
-    systemProperty "ide.mac.file.chooser.native", "false"
-    systemProperty "jbScreenMenuBar.enabled", "false"
-    systemProperty "apple.laf.useScreenMenuBar", "false"
+intellijPlatformTesting.runIde {
+    runIdeForUiTests {
+        task {
+            jvmArgumentProviders.add({
+                [
+                        "-Drobot-server.port=8082",
```

Comment:
Why -D added before each systemProperty

Reply:
Hi @dessina-devasia, thanks for reviewing , As part of migrating the `Gradle IntelliJ plugin` from version `1.*` to `2.*`, JetBrains suggested this change. You can refer to the documentation [here](https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin-migration.html#runideforuitests).

I tested by removing `-D`, but I encountered a build failure when running UI tests.

<img width="1531" alt="Screenshot 2024-11-18 at 1 37 41 PM" src="https://github.com/user-attachments/assets/2be70d03-bd4e-48a6-84e9-fc7b38f7ce45">

So we can't remove `-D`.
Thanks

Reply:
@vaisakhkannan Thanks for the clarification